lib/docker/util.rb

```diff
@@ -1,8 +1,10 @@
+require 'set'
+
 # This module holds shared logic that doesn't really belong anywhere else in the
 # gem.
 module Docker::Util
   # http://www.tldp.org/LDP/GNU-Linux-Tools-Summary/html/x11655.htm#STANDARD-WILDCARDS
-  GLOB_WILDCARDS = /[\?\*\[\{]/
+  GLOB_WILDCARDS = /[\?\*\[\{\]\}]/
```

Comment:
This is no longer used anywhere (I factored out the need for it while reducing ABC complexity), but I wasn't sure whether to delete it

Reply:
That makes sense. Let's leave it, but maybe we can add a comment that it's unused now?